### DISCUSSION TITLE
Сброс значений  ручного ввода

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#508] [Как это сделать?] [answered]
Добрый день!
Подскажите пожалуйста кто знает, как сделать. Есть столбец с расчетным значением, но в части ячеек оно перекрыто значением "Ручной ввод". Как по всему столбцу разом сбросить значения к расчетным?

**Comment by erpic-ru** [comment, marked as answer]
https://ru.docs.totum.online/functions#fn-clearList

**Reply by Spyker086** [reply]
СПАСИБО!!!
